src/main/java/duke/Event.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+public class Event extends Task{
+    protected String place;
+
+    /**
+     * constructor for Event
+     * @param d task description
```

Comment:
```suggestion
     * constructor for Event
     *
     * @param d task description
```
Don't forget your space!